src/filter/owner.rs

```diff
@@ -134,6 +147,10 @@ mod owner_parsing {
         both_negate:"!4:!3" => Ok(OwnerFilter { uid: NotEq(4), gid: NotEq(3)   }),
         uid_not_gid:"6:!8"  => Ok(OwnerFilter { uid: Equal(6), gid: NotEq(8)   }),
 
+        orphan_uid: "orphan"       => Ok(OwnerFilter { uid: Orphaned, gid: Ignore   }),
```

Comment:
It is possible that a user or group is actually named "orphan". I think it would probably be better to use a a different option, or maybe use something that isn't a valid username like "-".